All our architectures have IEEE754 floating point, build the same set of
features on all of them.

Index: configure
--- configure.orig
+++ configure
@@ -2960,7 +2960,7 @@ printf "%s\n" "netbsd" >&6; }
 # OpenBSD.
 #
 
-*86-*-openbsd* | *86_64-*-openbsd*)
+*-openbsd*)
 
 printf "%s\n" "#define BUILD_BSD 1" >>confdefs.h
 
